templates/app/maintenance-overview.phtml

```diff
@@ -35,7 +37,9 @@ $this->end();
             The page is automatically refreshed daily and is publicly available.
         </p>
         <p>
-            There is also a JSON version <a href="/share/properties.json" target="_blank">here</a>. It contains the full list of properties for each package. Note that the properties differ between organizations.
+            There is also a JSON version
+            <a href="<?= $this->url('app.maintenance-status') ?>" target="_blank">here</a>.
+            It contains the full list of properties for each package. Note that the properties differ between organizations.
```

Comment:
I would suggest moving this to the bottom of the page as well as the information on embedding dynamic JSON badges.

Reply:
well, if we move it to the botton of the page, i think nobody will read it, it is a long page ...
Let' s keep it that way

Reply:
My idea was: for a secondary information - which it is - it's fine.

Reply:
I moved that info about json file at the bottom of the page